schema.graphql

```diff
@@ -831,6 +856,37 @@ type Subscription {
   onPluginUpdate: Plugin
 }
 
+type Task {
+  _id: ID!
+  completed: Boolean
+  createdAt: DateTime!
+  creator: User!
```

Comment:
@duplixx 

1. We removed Projects and Tasks from the API a few PRs ago.  Your starting code is out of date
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1627
1. This is going to be replaced with Action Items.
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1648
1. Remove this Type and all references to it